doc/api/http2.md

```diff
@@ -320,6 +320,17 @@ added: v9.4.0
 Will be `true` if this `Http2Session` instance has been closed, otherwise
 `false`.
 
+#### http2session.connecting
+<!-- YAML
+added: v9.12.0
+-->
+
+* Value: {boolean}
```

Comment:
Just a hint: the `Value` will probably be removed soon. See https://github.com/nodejs/node/pull/19869. If that lands before this PR, this should get a update and vice versa.

Reply:
Update: it just landed and now the `Value` should be removed ;-)

Reply:
It seems this is a new section, so no conflicts for now, but the `* Value: {boolean}` should b replaced with just `* {boolean}`.

Reply:
Fixed in 8316d4f36b769fdf3224b65fdd32f4194a441ada